client/state/sites/actions.js

```diff
@@ -119,23 +119,29 @@ export function requestSite( siteId ) {
 			siteId,
 		} );
 
-		return wpcom
-			.site( siteId )
-			.get()
-			.then( site => {
-				dispatch( receiveSite( omit( site, '_headers' ) ) );
-				dispatch( {
-					type: SITE_REQUEST_SUCCESS,
-					siteId,
-				} );
-			} )
-			.catch( error => {
-				dispatch( {
-					type: SITE_REQUEST_FAILURE,
-					siteId,
-					error,
+		return new Promise( ( resolve, reject ) => {
+			wpcom
+				.site( siteId )
```

Comment:
Will this work even if `siteId` is a site fragment? If yes, the parameter should be renamed to `siteFragment`.

Reply:
I'll have to rename it on all places, but will do.

Reply:
Actually, it's quite a lot of places (reducers, selectors, actions and their usages for both wp.com and Jetpack sites). Do you think it's worth the renaming effort? Or is it enough to just rename the param to the `requestSite` fn and then dispatch actions as `siteId: siteFragment`?

Reply:
I was suggesting a rename just in this one function. I agree that renaming everything to `siteIdOrFragment` is overkill.